### DISCUSSION TITLE
Google System Method

### DISCUSSION THREAD

**Discussion by S-PScripts** [3kh0/ext-remover#718] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
Ok, this one should probably have something like "Might Not Work"

This is because it gives the following when trying to put the "google@d11.org" in the text box:
This account is not allowed to sign in within this network. Please talk to your network administrator for more information.
[This happens with any email apart from ones from my school.]

**Comment by replays36DXD** [comment]
what is this

**Reply by replays36DXD** [reply]
it just sends me to g,mail and makes me send an email to google@d11.org

**Comment by RiptideMC** [comment]
lol imageine

**Reply by S-PScripts** [reply]
sybau
